Makefile

```diff
@@ -83,7 +83,34 @@ fallthrough: submodules
 manifests:
 	@$(WARN) Deprecated. Please run make generate instead.
 
-generate: $(KUSTOMIZE) go.generate manifests.annotate
+generate: $(KUSTOMIZE) go.generate manifests.prepare manifests.annotate
+
+
+CROSSPLANE_TYPES_CHART_DIR = cluster/charts/crossplane-types
+CROSSPLANE_CONTROLLERS_CHART_DIR = cluster/charts/crossplane-controllers
+CROSSPLANE_CHART_DIR = cluster/charts/crossplane
+
+CRD_DIR = $(CROSSPLANE_TYPES_CHART_DIR)/crds
+CROSSPLANE_CHART_HELM2_CRD_DIR = $(CROSSPLANE_CHART_DIR)/templates/crds
+CROSSPLANE_CHART_HELM3_CRD_DIR = $(CROSSPLANE_CHART_DIR)/crds
+
+TYPE_MANIFESTS = clusterrole.yaml clusterrolebinding.yaml crossplane-admin-clusterrole.yaml \
```

Comment:
would a `*.yaml` pattern work here instead of having to explicitly list each file name? i can see this being tricky to maintain or figure out in the future first that our helm chart is missing a new file then figuring out why.

Reply:
Agreed, done.

Reply:
I also added a simple readme file as a note for working on helm charts.